### PR TITLE
Merge any column #28

### PR DESCRIPTION
**Pull request by rozetko**
Issue: https://github.com/CorpGlory/csv2influx/issues/28
Implemented merge feature, added tests, updated docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,9 @@ See more [examples](examples) to learn more about mapping
 
 ## Changelog
 
+###### 0.0.13
```
Comment (jonyrock) [inline README.md]: why new version? We haven't made release yet

File: examples/traffic_violations.conf.json [hunk 2]
```diff
@@ -8,8 +8,8 @@
       "format": "jsDate"
     },
     "fieldSchema": {
-      "description": {
-        "from": "Description",
+      "coordinates": {
+        "from": "${Latitude},${Longitude}",
```
Comment (jonyrock) [inline examples/traffic_violations.conf.json:12]: awesome

File: test/importer.js [hunk 3]
```diff
@@ -35,6 +39,17 @@ describe("Importer", function() {
         }
       }
     };
+
+    var record = {
```
Comment (jonyrock) [inline test/importer.js:43]: haha

File: README.md (hunk 1 above)
Comment (jonyrock) [reply README.md]: let's keep 0.0.12  until we make release.

File: importer.js [hunk 4]
```diff
@@ -134,7 +153,16 @@ class Importer {
...
         } else {
-          this._checkColInCols(this.fieldsNamesMapping[key], cols);
+          var tpl = new template.Template(this.fieldsNamesMapping[key]);
+          var items = tpl.getItems();
+          if(items.length > 0) {
+            items.forEach(
+              (item) => this._checkColInCols(item, cols)
+            );
+          }
+          else {
```
Comment (jonyrock) [inline importer.js]: `} else { ` , come on! :)

**Review by jonyrock — COMMENTED** [review]
I asked only for small fixes here

File: importer.js (hunk 4 above)
Comment (rozetko) [reply importer.js]: oops

**Review by jonyrock — APPROVED** [review]
cool

File: test/importer.js [hunk 5]
```diff
@@ -80,5 +81,23 @@ describe("Importer", function() {
         'address': 'Nevskiy pr.\nSt. Petersburg,Russia\n192121'
       });
     });    
+=======
```
Comment (jonyrock) [inline test/importer.js]: WTF?!

Comment (rozetko) [reply test/importer.js]: Oops, seems like bad merge.

**Review by jonyrock — APPROVED** [review]
cool

**Pull request merged**